core/services/gateway/handlers/webapicapabilities/webapi.go

```diff
@@ -15,3 +19,52 @@ type TargetResponsePayload struct {
 	Headers      map[string]string `json:"headers,omitempty"`       // HTTP headers
 	Body         []byte            `json:"body,omitempty"`          // HTTP response body
 }
+
+const TriggerType = "web-trigger@1.0.0"
```

Comment:
This should live within the web-trigger itself, since the only place it is used is there and in tests.